plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -60,7 +60,7 @@ type authorizingVisitor struct {
 	errors  []error
 }
 
-func (v *authorizingVisitor) visit(source fmt.Stringer, rule *rbacv1.PolicyRule, err error) bool {
+func (v *authorizingVisitor) visit(ctx context.Context, source fmt.Stringer, rule *rbacv1.PolicyRule, err error) bool {
```

Comment:
> I didn't find traces where it is used below but I can add 👍

These need to check the context for cancelation and terminate the visitor early in that case.

Reply:
@enj addressed in latest commit https://github.com/kubernetes/kubernetes/pull/113971/commits/800bed1cec1e1c0b58d5382fd85f883d7bb4a5b0, could you PTAL?